include/dcmqi/MultiframeConverter.h

```diff
@@ -35,15 +35,21 @@
 
 using namespace std;
 
+// common type definitions
 typedef short ShortPixelType;
 typedef itk::Image<ShortPixelType, 3> ShortImageType;
 typedef itk::ImageFileReader<ShortImageType> ShortReaderType;
 
 namespace dcmqi {
 
-  class ConverterBase {
+  class MultiframeConverter {
```

Comment:
The filenames use "Multiframe" but some of the code uses "MultiFrame" so it should be uniform in the codebase.

The DICOM standard uses "Multi-frame" which can't be used as an identifier, so that doesn't help.

It looks like DCMTK has a mix of MultiFrame and Multiframe.  

Personally I prefer Multiframe and would suggest using that consistently and that looks like Andrey's preference too.